py/parse_and_notify.py

```diff
@@ -134,53 +138,46 @@ def build(cls, raw_content: Mapping[str, Any]) -> UpdatedTopic:
 @dataclass(frozen=True)
 class TopicctlOutput:
     dry_run: bool
-    topics: Sequence[Topic]
+    topic: Topic
 
     @classmethod
     def build(cls, raw_content: str) -> TopicctlOutput:
         parsed = json.loads(raw_content)
         return TopicctlOutput(
             dry_run=parsed["dryRun"],
-            topics=[
-                *[
-                    NewTopic.build(content)
-                    for content in parsed["newTopics"] or []
-                ],
-                *[
-                    UpdatedTopic.build(content)
-                    for content in parsed["updatedTopics"] or []
-                ],
-            ],
+            topic=NewTopic.build(parsed["newTopic"])
+            if parsed["newTopic"]
+            else UpdatedTopic.build(parsed["updatedTopic"]),
         )
 
 
 def main():
-    input_data = sys.stdin.read()
-    topics = TopicctlOutput.build(input_data)
-
     token = os.getenv("DATADOG_API_KEY")
     assert token is not None, "No Datadog token in DATADOG_API_KEY env var"
-    notifier = Notifier(datadog_api_key=token)
-
-    dry_run = "Dry run: " if topics.dry_run else ""
-    tags = {
-        "source": "topicctl",
-        "source_category": "infra_tools",
-        "sentry_region": SENTRY_REGION,
-    }
+    input_data = sys.stdin.readlines()
```

Comment:
using `readlines()` over `read()` and looping over that fixes the issue where only one topic could be applied at a time